inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -277,6 +278,25 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
                 return node->input_value(0).get_partial_shape().rank().get_length() > 5;
             });
 
+    auto normalizeL2FusionCallback = [](const_node_ptr &node) -> bool {
```

Comment:
Actually `isSupportedOperation` was designed to provide an ability to check if the ngraph operation is supported by the plug-in or not. Besides other usecases the intention was to simplify transformation pipeline adjustments (by reusing  `isSupportedOperation` inside calls callbacks). 
In current Normalize fusion implementations `transformation_callback` is called with constant node (not resulting normalize) which makes impossible reuse of `isSupportedOperation`.
So my question is: does is makes sense to change fusing transformations design in order to call callback with resulting node? I understand there might be negative impacts, but still want to discuss the topic from general design perspective.

cc'ed @GlebKazantaev